.github/workflows/check.yml

```diff
@@ -10,8 +10,6 @@ jobs:
     strategy:
       matrix:
         version:
-          - zeek:5.0.4
-          - zeek:5.1.1
```

Comment:
Since we do not publish to `zeekurity` anymore, this will not get you the latest, you also need to change `zeekurity` to `zeek` on line 20 below.

Reply:
Resolving since zeek/zeek publishes to `zeekurity` again in addition to `zeek`.

Reply:
This works now. I'll wait for upcoming 5.2 RC and then add that back in before finalizing the PR.